tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/KeycloakRealmImportOperatorProvisioner.java

```diff
@@ -0,0 +1,350 @@
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.assertj.core.util.Lists;
+import org.assertj.core.util.Strings;
+import org.jboss.intersmash.tools.IntersmashConfig;
+import org.jboss.intersmash.tools.application.openshift.KeycloakRealmImportOperatorApplication;
+import org.jboss.intersmash.tools.provision.openshift.operator.OperatorProvisioner;
+import org.jboss.intersmash.tools.util.tls.CertificatesUtils;
+import org.keycloak.k8s.v2alpha1.Keycloak;
+import org.keycloak.k8s.v2alpha1.KeycloakRealmImport;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Http;
+import org.slf4j.event.Level;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.event.helpers.EventHelper;
+import cz.xtf.core.openshift.OpenShiftWaiters;
+import cz.xtf.core.waiting.SimpleWaiter;
+import cz.xtf.core.waiting.failfast.FailFastCheck;
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.api.model.KubernetesResourceList;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.apps.StatefulSet;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.NonNamespaceOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.NonNull;
+
+/**
+ * Keycloak operator provisioner
+ */
+public class KeycloakRealmImportOperatorProvisioner extends OperatorProvisioner<KeycloakRealmImportOperatorApplication> {
+
+	//private MixedOperation<KeycloakRealmImport, KubernetesResourceList<KeycloakRealmImport>, Resource<KeycloakRealmImport>> keycloakRealmImportClient;
+	//private MixedOperation<Keycloak, KubernetesResourceList<Keycloak>, Resource<Keycloak>> keycloakClient;
+	private static final String OPERATOR_ID = IntersmashConfig.keycloakQuarkusOperatorPackageManifest();
+	protected FailFastCheck ffCheck = () -> false;
+
+	public KeycloakRealmImportOperatorProvisioner(@NonNull KeycloakRealmImportOperatorApplication application) {
+		super(application, OPERATOR_ID);
+	}
+
+	public static String getOperatorId() {
+		return OPERATOR_ID;
+	}
+
+	@Override
+	protected String getOperatorCatalogSource() {
+		return IntersmashConfig.keycloakQuarkusOperatorCatalogSource();
+	}
+
+	@Override
+	protected String getOperatorIndexImage() {
+		return IntersmashConfig.keycloakQuarkusOperatorIndexImage();
+	}
+
+	@Override
+	protected String getOperatorChannel() {
+		return IntersmashConfig.keycloakQuarkusOperatorChannel();
+	}
+
+	@Override
+	public void subscribe() {
+		if (Strings.isNullOrEmpty(IntersmashConfig.keycloakQuarkusImageURL())) {
+			super.subscribe();
+		} else {
+			// RELATED_IMAGE_RHSSO_OPENJ9 and RELATED_IMAGE_RHSSO_OPENJDK, determine the final value for RELATED_IMAGE_RHSSO
+			subscribe(
+					INSTALLPLAN_APPROVAL_MANUAL,
+					// TODO: check if these env variables still make sense in the new quarkus operator
```

Comment:
`TODO` is to be solved or should we keep it in here for some reason?

Reply:
removed